Enumerables.rb

```diff
@@ -0,0 +1,70 @@
+module Enumerable
+    def my_each
+        i = 0
+        while i<self.size
+            yield self[i]
+            i+=1
+        end
+    end
+
+    def my_each_with_indeitem
+        i = 0
+        result = []
+        while i < self.size
+            result << yield(self[i],i)
+            i +=1
+        end
+        result
+    end
+
+    def my_select
+        result=[]
+        my_each {|item| result<<item if yield item}
+        result
+    end
+
+    def my_all?
+        my_each{|item| return false unless yield item}
+        true
+    end
+
+    def my_any?
+        my_each{|item| return true if yield item}
+        false
+    end
+
+    def my_none
+        my_each{|item| return false if yield item}
+        true
```

Comment:
- **[OPTIONAL]** It's a common convention in ruby that any methods return `true` or `false` to end with question mark `?`